mypy/typeshed/stdlib/turtle.pyi

```diff
@@ -671,7 +671,7 @@ def shearfactor(shear: float) -> None: ...
 
 # Unsafely overlaps when no arguments are provided
 @overload
-def shapetransform() -> tuple[float, float, float, float]: ...  # type: ignore[overload-overlap]
```

Comment:
What causes all these changes? Is this because of the `MutableMapping` overlap logic?

Reply:
Sorry, my debug code 🤦‍♂️